tests/wait_for_services.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from retrying import retry
+from requests import RequestException
+
+from tests.config import Config
+
+
+class HealthCheckException(Exception):
+    def __init__(self, service):
+        self.service = service
+
+    def __str__(self) -> str:
+        return f'Healthcheck fails for {self.service.lower()}'
+
+
+def retry_if_http_error(exception):
+    print(f'error has occurred: {str(exception)}')
+    return isinstance(exception, RequestException) or isinstance(exception, HealthCheckException)
+
+
+@retry(retry_on_exception=retry_if_http_error, wait_fixed=10000, stop_max_delay=600000, wrap_exception=True)
```

Comment:
Shouldn't we also add a maximum number of retries before ceasing?

Reply:
@damien-lloyd `stop_max_delay=600000`.